http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/KubernetesServiceInterp.scala

```diff
@@ -297,7 +303,10 @@ final class LeoKubernetesServiceInterp[F[_]: Parallel](
       }
       // create list of Precreating nodepools
       eitherNodepoolsOrError = List.tabulate(req.numNodepools.value) { _ =>
-        getUserNodepool(clusterId, userInfo, req.kubernetesRuntimeConfig, ctx.now)
+        getUserNodepool(clusterId,
+                        userInfo.copy(userEmail = WorkbenchEmail("nouserassigned@gmail.com")),
```

Comment:
we cannot use the email of the person issuing the batch creation request here, because claiming must occur before a user can be assigned to it.